qemu/tests/cfg/numa_opts.cfg

```diff
@@ -101,11 +101,17 @@
             numa_expected = "0-63"
         - nodes.128:
             no aarch64
+            x86_64:
+                no RHEL.9.2
```

Comment:
Is this strategy for x86_64 only ?  From the issue,  it is all platform if i understand correctly.  @PaulYuuu @zhenyzha

Reply:
@Please check 2041323#c4 for details

Reply:
I missed `no aarch64` for this case.  sorry, please ignore this comment, thanks!